src/sidebar/components/LaunchErrorPanel.js

```diff
@@ -16,9 +16,7 @@ import Panel from './Panel';
 export default function LaunchErrorPanel({ error }) {
   return (
     <div className="LaunchErrorPanel">
-      <Panel title="Unable to start Hypothesis">
-        <p>{error.message}</p>
-      </Panel>
+      <Panel title="Unable to start Hypothesis">{error.message}</Panel>
```

Comment:
The children used to be wrapped in a `<p>` but no longer are. Was that intended?

Reply:
Dratted element classes biting again. The global `margin` set on `p` made for too much space inside this box. Needs better fixing. Should I note it?

Reply:
To summarize what I can see after comparing this and other panels:

Most of the other panels in the app that have some explanatory text near the top have that text wrapped in a `<p>` element. Whether the text is wrapped in a `<p>` element does not affect the margin above the text, but for this panel does create some extra space between the bottom of the paragraph and the bottom of the dialog. In the context of other dialogs this does not happen because there happens to be non-paragraph content below the paragraph. For example the "Annotation Unavailable" panel has a "Show all annotations" button at the bottom.

This issue could affect other panels in future where the last child element is a paragraph. Perhaps a generic solution would be appropriate, like:

```html
<div class="Hyp-Panel">
  <header><h2 class="Hyp-Panel__title">Unable to start Hypothesis</h2></header>
  <div class="Hyp-Panel__content">
    <p>Oh no!</p>
  </div>
</div>
```

```css
.Hyp-Panel__content > p:last-child {
  margin-bottom: 0;
}
```

Reply:
I'd like to solve this by simplification versus additional rules. I don't think that `p` elements should have element styles. I'm going to look at this as part of https://github.com/hypothesis/frontend-shared/issues/97